src/client.c

```diff
@@ -66,13 +88,23 @@ int send_request(int fd, char *hostname, char *port, char *path)
 {
   const int max_request_size = 16384;
   char request[max_request_size];
-  int rv;
 
-  ///////////////////
-  // IMPLEMENT ME! //
-  ///////////////////
+  int rv = recv(fd, request, max_request_size -1, 0);
```

Comment:
We shouldn't be calling `recv` here since this function only deals with sending an HTTP request.